### DISCUSSION TITLE
How to manually disconnect chromedriver before page load, and manually connect it after page loaded?

### DISCUSSION THREAD

**Discussion by iamumairayub** [seleniumbase/SeleniumBase#2315] [Q&A] [answered]
driver.uc_open_with_reconnect("https://pixelscan.net/", reconnect_time=60) basically locks our code for 60 seconds.

I am looking for something so we disconnect chromedriver first, then load webpage, once its loaded, we reconnect it.

I digged through the code but could not create a workable example.

```
url = "https://pixelscan.net/"

driver = Driver(
    uc=True,
    proxy='http://proxy'
    )

chromedriver.stop()
driver.get(url )
chromedriver.start()
```

**Comment by mdmintz** [comment, marked as answer]
You can’t disconnect chromedriver from Chrome before issuing the command to load the page because you still need to be connected in order to issue commands to Chrome. While disconnected, you cannot know when a page is done loading, because again, the connection is needed to know that. Therefore, a predetermined time is used for the disconnect/reconnect time, which is customizable.
